spinalcordtoolbox/scripts/sct_label_vertebrae.py

```diff
@@ -313,10 +335,11 @@ def main(argv=None):
 
     # resample to 0.5mm isotropic to match template resolution
     printv('\nResample to 0.5mm isotropic...', verbose)
-    s, o = run_proc(['sct_resample', '-i', 'data_straight.nii', '-mm', '0.5x0.5x0.5', '-x', 'linear', '-o', 'data_straightr.nii'], verbose=verbose)
+    s, o = run_proc(['sct_resample', '-i', 'data_straight.nii', '-mm', '0.5x0.5x0.5', '-x', 'linear', '-o',
+                     'data_straightr.nii'], verbose=verbose)
```

Comment:
`run_proc` -> `spinalcordtoolbox.resampling.resample_file`

Reply:
done in 8d4cf72